ts/components/wallet/PaymentMethodsList.tsx

```diff
@@ -128,11 +132,11 @@ class PaymentMethodsList extends React.Component<Props, never> {
             const disabledStyle = isItemDisabled ? styles.disabled : {};
             return (
               <ListItem
-                style={[styles.listItem]}
+                style={styles.listItem}
                 onPress={itemInfo.item.onPress}
                 underlayColor={underlayColor}
               >
-                <Left>
```

Comment:
These changes solve for iPhone 5s but they change the look for other devices


### iPhone X
| before  | now |
| ------------- | ------------- |
![before](https://user-images.githubusercontent.com/822471/76336494-5e317d80-62f6-11ea-97e9-0f06a8b73397.png) | ![now](https://user-images.githubusercontent.com/822471/76336512-67bae580-62f6-11ea-9deb-2583afbb6b0f.png)

Reply:
I thought it was a good compromise :sweat_smile: , I change it